dockerfiles/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM openjdk:8
+ARG HIVE_VERSION=3.0.0.13.3
+ARG HADOOP_VERSION_EE=3.2.0.12-EE-SNAPSHOT
+ARG PROMETHEUS_EXPORTER_VERSION="0.12.0"
+ARG USER="hive"
+ARG GROUP="hive"
+
+# This is for local dev in the VM
+RUN groupadd -g 1505 $GROUP && \
+    useradd -m --uid 1515 --gid 1505 $USER
+RUN groupadd -g 1511 metastore && \
+    usermod -a -G metastore $USER
+RUN groupadd -g 1234 hadoop && \
+    usermod -a -G hadoop $USER
+    
+
+
+# Download HADOOP
+RUN --mount=type=secret,id=wgetrc,dst=/root/.wgetrc \
+    mkdir -p /srv/hops && \
+    wget https://nexus.hops.works/repository/hopshadoop/hops-$HADOOP_VERSION_EE.tgz && \
+    tar -C /srv/hops/ -zxf hops-$HADOOP_VERSION_EE.tgz && \
+    ln -s /srv/hops/hadoop-$HADOOP_VERSION_EE /srv/hops/hadoop && \
+    rm -rf /srv/hops/hadoop/etc && \
+    rm -rf /srv/hops/hadoop/sbin && \
+    rm hops-$HADOOP_VERSION_EE.tgz
+
+# Make sure you build with --platform linux/amd64
+
+# /usr/local/openjdk-8/bin/java
+ENV JAVA_HOME=/usr/local/openjdk-8
+RUN mkdir -p /usr/lib/jvm &&  ln -s $JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+ENV HADOOP_HOME=/srv/hops/hadoop
+ENV HADOOP_CONF_DIR="$HADOOP_HOME/conf"
+
+# Download hive
+# https://archiva.hops.works/repository/Hops/io/hops/hive/hive-packaging/3.0.0.13.3/hive-packaging-3.0.0.13.3-bin.tar.gz
+RUN wget https://archiva.hops.works/repository/Hops/io/hops/hive/hive-packaging/$HIVE_VERSION/hive-packaging-$HIVE_VERSION-bin.tar.gz && \
+    tar -C /srv/hops/ -zxf hive-packaging-$HIVE_VERSION-bin.tar.gz && \
+    ln -s /srv/hops/apache-hive-$HIVE_VERSION-bin /srv/hops/hive && \
+    rm hive-packaging-$HIVE_VERSION-bin.tar.gz
+
+
+ENV HIVE_HOME=/srv/hops/hive
+ENV PATH="/srv/hops/super_crypto/hive:$HIVE_HOME/bin:/opt/venv/bin:$HADOOP_HOME/bin:$PATH"
+
+
+# Download extra jars
+# FIXME: this should come from our own repo
+ENV HOPSWORKS_JARS=/srv/hops/hopsworks-jars
+RUN mkdir -p ${HOPSWORKS_JARS}
+ARG PROMETHEUS_EXPORTER_VERSION="0.12.0"
+RUN wget -O ${HOPSWORKS_JARS}/jmx_prometheus_javaagent-${PROMETHEUS_EXPORTER_VERSION}.jar  https://repo.maven.apache.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/${PROMETHEUS_EXPORTER_VERSION}/jmx_prometheus_javaagent-${PROMETHEUS_EXPORTER_VERSION}.jar
+
+ARG MYSQL_CONNECTOR_VERSION="8.0.21"
+RUN wget -O ${HOPSWORKS_JARS}/mysql-connector-java-${MYSQL_CONNECTOR_VERSION}-bin.jar  https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.21/mysql-connector-java-8.0.21.jar
```

Comment:
we also have it on repo.hops.works
https://repo.hops.works/master/mysql-connector-java-8.0.21-bin.jar

Reply:
Fixed